pkg/daemon/ceph/osd/volume.go

```diff
@@ -76,7 +76,7 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 
 	batchArgs := append(baseArgs, []string{
 		osdsPerDeviceFlag,
-		strconv.Itoa(a.storeConfig.OSDsPerDevice),
+		convertOSDsPerDevice(a.storeConfig.OSDsPerDevice),
```

Comment:
It could be unclear to what `OSDsPerDevice` is being converted. I'm not entirely sure what would be more explicit. Perhaps `sanitizeOSDsPerDevice`?